core/src/main/java/org/mskcc/cbio/oncokb/util/AnnotationSearchUtils.java

```diff
@@ -0,0 +1,708 @@
+package org.mskcc.cbio.oncokb.util;
+
+import static org.mskcc.cbio.oncokb.Constants.DEFAULT_REFERENCE_GENOME;
+import static org.mskcc.cbio.oncokb.util.AlterationUtils.GENOMIC_CHANGE_FORMAT;
+import static org.mskcc.cbio.oncokb.util.AlterationUtils.HGVSG_FORMAT;
+import static org.mskcc.cbio.oncokb.util.LevelUtils.THERAPEUTIC_RESISTANCE_LEVELS;
+import static org.mskcc.cbio.oncokb.util.LevelUtils.THERAPEUTIC_SENSITIVE_LEVELS;
+
+import java.util.*;
+import java.util.regex.Matcher;
+import java.util.stream.Collectors;
+import org.apache.commons.collections.CollectionUtils;
+import org.apache.commons.collections.map.HashedMap;
+import org.apache.commons.lang3.StringUtils;
+import org.mskcc.cbio.oncokb.apiModels.CancerTypeMatch;
+import org.mskcc.cbio.oncokb.apiModels.DrugMatch;
+import org.mskcc.cbio.oncokb.apiModels.LevelsOfEvidenceMatch;
+import org.mskcc.cbio.oncokb.genomenexus.GNVariantAnnotationType;
+import org.mskcc.cbio.oncokb.model.*;
+
+public class AnnotationSearchUtils {
```

Comment:
No need to review this file. All copied from the original AnnotationsApiController so I can test some logics.